js/exporter/jspdf/v3/pdf_utils_v3.js

```diff
@@ -5,22 +5,26 @@ function round(value) {
     return Math.round(value * 1000) / 1000; // checked with browser zoom - 500%
 }
 
-function getTextLines(doc, text, font, { wordWrapEnabled, columnWidth }) {
+function getTextLines(doc, text, font, { wordWrapEnabled, textWidth }) {
     if(wordWrapEnabled) {
         // it also splits text by '\n' automatically
-        return doc.splitTextToSize(text, columnWidth, {
+        return doc.splitTextToSize(text, textWidth, {
             fontSize: font?.size || doc.getFontSize()
         });
     }
     return text.split('\n');
 }
 
-function calculateTextHeight(doc, text, font, { wordWrapEnabled, columnWidth }) {
+function calculateTextWidth(columnWidth, padding) {
```

Comment:
It doesn't calculate 'width of text'
It calculates 'content area width' (similar to https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Introduction_to_the_CSS_box_model)
calculateContentAreaWidth ?